src/components/ClimbDropdown.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-menu class="mr-3">
+    <template v-slot:activator="{ on }">
+      <v-btn
+        :color="states[route_state].color"
+        class="white--text"
+        v-on="on"
+      >
+        {{ states[route_state].abbrev }}
+      </v-btn>
+    </template>
+    <v-list>
+      <v-list-tile
+        v-for="state in orderedStates"
+        :key="state"
+        @click="route_state = state"
+      >
+        <v-list-tile-title>{{ states[state].name }}</v-list-tile-title>
+      </v-list-tile>
+    </v-list>
+  </v-menu>
+</template>
+
+<script>
+import { ClimbState } from './ClimbState.js'
+
+export default {
+  props: ['route_state'],
+  data: () => ({
+    states: [
```

Comment:
it was an existing problem, but i wonder if there's a better way to supply this data. right now, we're creating `states` and `orderedStates` instance variables, even though this is more like constant data that we want to share across all instances of the ClimbDropdown component.

(or is it? we'll probably want to be able to pass in different colors for the left and right buttons. the `name` and `abbrev` fields and `orderedStates` are constant, though.)